lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -197,7 +206,8 @@ sub web(Str $cfg-filename, Str $model-filename, Str $tech-file?) is export {
     }
 
     my $model = timed "Load model from $module-path/$module.nhd", {
-        load-model-using-cache($*HOME.add('.agrammon'), $module-path, $module)
+        my %preprocessor-options = set($variants.split(","));
```

Comment:
I do wonder if this line is repeated enough to be worth pulling the code out into a sub.

Reply:
Done.